aws_quickstart/main_v2.yaml

```diff
@@ -81,6 +105,17 @@ Conditions:
     Fn::Equals:
       - !Ref CloudSecurityPostureManagement
       - true
+  EnableAgentlessScanning:
```

Comment:
We probably need to enforce that if `EnableAgentlessScanning` is true, `CloudSecurityPostureManagement` is also true

Reply:
Updated!